### PR TITLE
React todo app

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi There!

I have made the following to the code:
- Initialize a Vite project :zap:
- Add Linters configuration :memo:
- Create `main.jsx`  :rocket:
- Create all the JSX components that the code needs to work :package:
- Add styling to the app :art:
- Add LICENSE and README :pencil2:

Thank you for taking the time to review my code!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React</title>
```
Comment (jlvFlores) [inline index.html:7]: **[OPTIONAL]** Make sure to change the title of your future Vite apps. 👍

File: src/todo
Comment (jlvFlores) [inline src/todo:1]: Be sure to remove any unused or empty files, as they take up unwanted space. 🤔

File: vite.config.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+// eslint-disable-next-line import/no-extraneous-dependencies
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  resolve: {
+    alias: {
+      '@': Path2D.resolve(__dirname, './src'),
+    },
+  },
```
Comment (jlvFlores) [inline vite.config.js:12]: **Suggestion:** Remove this line of code as it impedes the app from running. 🤖

**Review by jlvFlores — COMMENTED** [review]
Pull Request Review

Hello @jaimevillegas 

Great job so far, I think there are a few things that can be improved, but overall good job on your site.

Rocky

### Highlights 🎯
The Pull Request follows GitFlow ✅
Your commits are descriptive ✅

### Required Changes ♻️
- None

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

*Please consider, checking the comments below.*

Cheers and Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question, so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise.

File: index.html (hunk 1 above)
Comment (jaimevillegas) [reply index.html:7]: Ok! I will change the title of my webpage!

File: vite.config.js (hunk 2 above)
Comment (jaimevillegas) [reply vite.config.js:12]: I will work on that! 👨🏻‍💻🤓

**Pull request merged**